recompose-composer/src/main/kotlin/recompose/composer/writer/ParameterValue.kt

```diff
@@ -49,6 +49,10 @@ internal sealed class ParameterValue {
     class DrawableValue(
         val drawable: Drawable
     ) : ParameterValue()
+
+    class IntegerValue(
+        val raw: Int
+    ) : ParameterValue()
```

Comment:
I think for this we could reuse `RawValue`, adding a second constructor that takes int and calls the main constructor converting it to `String`. In this situation we do not care about the type anymore and just need to write out the raw value. :)

Reply:
Yes, that's better.